AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -236,34 +236,34 @@
 "accountdetails.title.newaccount" = "New Account";
 "accountdetails.title.editingaccount" = "Editing Account";
 "accounttype.cloud" = "Alfresco in the Cloud";
-"accounttype.alfrescoServer" = "Alfresco Server";
+"accounttype.alfrescoServer" = "Alfresco One";
```

Comment:
In Android I noticed we refer to Alfresco One, so was aligning with that.

I *think* there was a discussion around Android, but your call if Server is preferable.